### PR TITLE
feat(Bilibili): support notifications

### PR DESCRIPTION
**Pull request by Fatpandac**
## Description

support notifications

## Screencast

image

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/Bilibili/package.json [hunk 1]
```diff
@@ -43,7 +50,8 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.39.3"
+    "@raycast/api": "^1.39.3",
```
Comment (mil3na) [inline extensions/Bilibili/package.json:53]: Do you mind updating to the latest Raycast API version?

Comment (Fatpandac) [reply extensions/Bilibili/package.json:53]: I do not mind.

**Comment by mil3na** [comment]
Is there any other way to login to your extension that is not by the QR code? I managed to create an account, but the app is not available in my App Store's region.

**Comment by Fatpandac** [comment]
There is no option to log in without using the QR code. Moreover, the Bilibili app is only available on the Chinese App Store.

**Comment by mil3na** [comment]
Okay, would it be possible for you to make a video of the change in action? Just to make sure we are not introducing something that doesn't work :)

**Comment by Fatpandac** [comment]
录屏2023-07-13 16 36 50

Check here.

**Pull request merged**